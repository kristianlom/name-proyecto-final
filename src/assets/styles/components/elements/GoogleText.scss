.GoogleText {
  font-size: 18px;
  font-family: 'Quicksand Bold';
  margin-left: 5px;
  &--blue {
    color: #4285F4;;
  }
  &--red {
    color:#EA4335;
  }
  &--yellow {
    color: #FBBC05;
  }

  &--green {
    color:#34A853;;
  }
}
