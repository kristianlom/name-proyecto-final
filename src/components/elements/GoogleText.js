import React from 'react';
import '../../assets/styles/components/elements/GoogleText.scss';

const GoogleText = () => (
  <p className="GoogleText">
    <span className="GoogleText--blue">G</span>
    <span className="GoogleText--red">o</span>
    <span className="GoogleText--yellow">o</span>
    <span className="GoogleText--blue">g</span>
    <span className="GoogleText--green">l</span>
    <span className="GoogleText--red">e</span>
  </p>
);

export default GoogleText;
