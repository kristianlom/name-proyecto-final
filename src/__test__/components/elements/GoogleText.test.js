import React from 'react';
import { shallow } from 'enzyme';
import GoogleText from '../../../components/elements/GoogleText';

describe('GoogleText', () => {
  it('should render GoogleText', () => {
    const component = shallow(<GoogleText />);
    expect(component).toMatchSnapshot();
  });
});

