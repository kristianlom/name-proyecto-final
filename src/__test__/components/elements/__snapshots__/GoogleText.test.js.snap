// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleText should render GoogleText 1`] = `ShallowWrapper {}`;
